src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -395,9 +395,7 @@ final class OnboardingInfoViewController: UIViewController {
 		case .enableLoggingOfContactsPage:
 			func handleBluetooth(completion: @escaping () -> Void) {
 				if let alertController = self.exposureManager.alertForBluetoothOff(completion: { completion() }) {
-					self.present(alertController, animated: true)
```

Comment:
Don't you think the `handleBluetooth` function is redundant then after this change? So why is this not removed at all and and removed from all the subsequent calls, like in `askExposureNotificationsPermissions` below?